Add information to docs (#10370)
qmk/qmk_firmware: docs/ref_functions.md

### `update_tri_layer_state(state, x, y, z)`
The other function is `update_tri_layer_state(state, x, y, z)`.  This function is meant to be called from the [`layer_state_set_*` functions](custom_quantum_functions.md#layer-change-code).  This means that any time that you use a keycode to change the layer, this will be checked.  So you could use `LT(layer, kc)` to change the layer and it will trigger the same layer check.

> The caveat to this method is that you cannot access the `z` layer without having `x` and `y` layers on, since if you try to activate just layer `z`, it will run this code and turn off layer `z` before you could use it. Additionally, layer `z` must be a higher layer than `x` and `y`.

Reviewer: See, I disagree, it works right, but it may produce unexpected behavior.   If the raise and lower layers have transparencies in the same spots, then the adjust layer may be useful there.   Admittedly, this is an edge case, but I think that it should still be considered. 

```suggestion
There are a couple of caveats to this method: 
1. You cannot access the `z` layer without having `x` and `y` layers on, since if you try to activate just layer `z`, it will run this code and turn off layer `z` before you could use it. 
2. Because layers are processed from the highest number `z` should be a higher layer than `x` and `y` or you may not be able to access it.
```

Author: Thanks for writing it better. I'm a bit inexperienced writing docs.

### `update_tri_layer_state(state, x, y, z)`
The other function is `update_tri_layer_state(state, x, y, z)`.  This function is meant to be called from the [`layer_state_set_*` functions](custom_quantum_functions.md#layer-change-code).  This means that any time that you use a keycode to change the layer, this will be checked.  So you could use `LT(layer, kc)` to change the layer and it will trigger the same layer check.

> There are a couple of caveats to this method: 

Reviewer 2: ```suggestion
There are a couple of caveats to this method:
```

Author: It's so hard to see extra whitespace on the website

Reviewer 2: [Refined GitHub](https://github.com/sindresorhus/refined-github) is a godsend :)

> 1. You cannot access the `z` layer without having `x` and `y` layers on, since if you try to activate just layer `z`, it will run this code and turn off layer `z` before you could use it. 

Reviewer 2: ```suggestion
1. You cannot access the `z` layer without having `x` and `y` layers on, since if you try to activate just layer `z`, it will run this code and turn off layer `z` before you could use it.
```
